modules/nf-core/dysgu/meta.yml

```diff
@@ -0,0 +1,78 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "dysgu"
+## TODO nf-core: Add a description of the module and list keywords
+description: Dysgu calls structural variants (SVs) from mapped sequencing reads. It is designed for accurate and efficient detection of structural variations.
+keywords:
+  - structural variants
+  - sv
+  - vcf
+tools:
+  - dysgu:
+      description: Structural variant caller for mapped sequencing data
+      homepage: https://github.com/kcleal/dysgu
+      documentation: https://github.com/kcleal/dysgu/blob/master/README.rst
+      tool_dev_url: https://github.com/kcleal/dysgu
+      doi: "10.1093/nar/gkac039"
+      licence: ["GPL-3.0-or-later"]
+version: "1.6.2"
```

Comment:
can the version be emitted by the tool itself?

Reply:
Yes, dysgu can emit the version by itself `dysgu --version > dysgu_version.txt` I will add this line of code to emit version.